checker.py

```diff
@@ -15,12 +15,15 @@
 from owscheck import OwsChecker
 from bypassSSLVerification import bypassSSLVerification
 
-# Logging configuration
-logger = logging.getLogger("owschecker")
-out_hdlr = logging.StreamHandler(sys.stdout)
-out_hdlr.setLevel(logging.INFO)
-logger.addHandler(out_hdlr)
-logger.setLevel(logging.INFO)
+
+def setLogger(filename):
```

Comment:
I'd better have the method named `setFileLogger`, since it should be possible to send logs to whatever you want (rsyslog, logstash, ...), and in your case, it's actually a file.

Anyway, is there any better mechanisms in python to configure logging (as in Java, `logstash.{xml,groovy}`, `log4j.{xml,properties}` ?). Ideally it should be configureable via a configuration file, instead of having the logic source code-side.

Looks good though